src/sentry/integrations/gitlab/integration.py

```diff
@@ -106,7 +116,7 @@ class InstallationForm(forms.Form):
     group = forms.CharField(
         label=_("GitLab Group Name"),
```

Comment:
This isn't part of this change, but is there any reason why there are double quotes here?

Reply:
They were there when I got here? I can change them now though.